beacon-chain/core/validators/slashing.go

```diff
@@ -0,0 +1,33 @@
+package validators
+
+import (
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/runtime/version"
+)
+
+// SlashingParamsPerVersion returns the slashing parameters for the given state version.
+func SlashingParamsPerVersion(v int) (slashingQuotient, proposerRewardQuotient, whistleblowerRewardQuotient uint64, err error) {
```

Comment:
It could be dangerous if someone messes up the order of the return variables. However, there are more dangerous things out there...